from_csv_to_json.py

```diff
@@ -0,0 +1,58 @@
+from converter_file_format import Converter
+from json import dumps
+
+class CSV_JSON(Converter):
+
+    def __init__(self, file_csv):
+        self.nmcsv = file_csv
+        self.data_csv = []          #храниние получаемой инфу из файла
+        self.open()                 #прочитать файл
+        self.data_json = {}         #запись в формате словаря
+        self.header_keys = []
+        self.read_header()
+
+    def close(self):
+        pass
+
+    def open(self):
+        with open(self.nmcsv, 'r') as fl:
+            for line in fl:
+                self.data_csv.append(line.strip('\n'))
+
+        print(self.data_csv)
```

Comment:
Should it really be here? If it does need to be here, you logging library instead of common print function